MapsetVerifier.Checks/Taiko/Timing/CheckKiaiConsistency.cs

```diff
@@ -0,0 +1,80 @@
+﻿using MapsetVerifier.Framework.Objects;
+using MapsetVerifier.Framework.Objects.Attributes;
+using MapsetVerifier.Framework.Objects.Metadata;
+using MapsetVerifier.Parser.Objects;
+
+using static MapsetVerifier.Checks.Utils.TaikoUtils;
+
+namespace MapsetVerifier.Checks.Taiko.Timing
+{
+    [Check]
+    public class CheckKiaiConsistency : GeneralCheck
+    {
+        public override CheckMetadata GetMetadata() =>
+            new BeatmapCheckMetadata()
+            {
+                Author = "SN707, Nostril",
+                Category = "Compose",
+                Message = "Kiai inconsistencies",
+                Modes = [Beatmap.Mode.Taiko],
```

Comment:
I'm not familiar enough with how Kiais tend to be approached in other modes, but would this check be possibly relevant in the other ones?